### PR TITLE
C164160483 edgetests

### PR DESCRIPTION
**Pull request by justincrosbiehaventec**

### CODE REVIEW

File: Example/Tests/HaventecAuthenticateTests.swift [hunk 1]
```diff
@@ -80,6 +80,29 @@ class HaventecAuthenticateTest: XCTestCase {
         XCTAssertThrowsError(try HaventecAuthenticate.getUserUuid())
     }
 
+    func testEdgeCases() throws {
```
Comment (htclifford) [inline Example/Tests/HaventecAuthenticateTests.swift:85]: I don't know what the edge case is, break down into specific test methods

Comment (htclifford) [inline Example/Tests/HaventecAuthenticateTests.swift:85]: Add edge test for:
- Hard coded hashPin that's of proper length but not valid value to any device (e.g. 'AAAA....')

Comment (justincrosbiehaventec) [reply Example/Tests/HaventecAuthenticateTests.swift:85]: Added extra base64 asserts

Comment (justincrosbiehaventec) [reply Example/Tests/HaventecAuthenticateTests.swift:85]: Multiple edge cases. The asserts are readable

File: HaventecAuthenticate.podspec [hunk 2]
```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
 
   s.source_files = 'HaventecAuthenticate/Classes/**/*'
   
-  s.dependency 'HaventecCommon'
+  s.dependency 'HaventecCommon', path => '/Users/justincrosbie/dev/haven/sdk/ios/common-ios-sdk'
```
Comment (htclifford) [inline HaventecAuthenticate.podspec]: Make sure to revert changes in the podspec
